ui/src/stores/eventConfigDetailStore.ts

```diff
@@ -49,6 +50,20 @@ export const useEventConfigDetailStore = defineStore('useEventConfigDetailStore'
     hideDeleteEventConfigEventModal() {
       this.deleteEventConfigEventModalState.visible = false
       this.deleteEventConfigEventModalState.eventConfigEvent = null
+    },
+    async disableEventConfigEvent(eventId: number) {
+      // Placeholder for disabling event logic
+      const response = await changeEventConfigEventStatus(eventId, this.selectedSource?.id || 0, false)
```

Comment:
Do we need an event source? If `this.selectedSource.id` is `0` or `undefined`, probably should not call the Rest API.